stream/src/js/stream.js

```diff
@@ -6,24 +6,26 @@ $(document).ready(function() {
     // I'll admit. Not the cleanest way but it'll work!
     $('#signup-form').submit(function(e) {
         var form = $(e.target);
-
-        var handler = function(form, success) {
-            if (success) {
-                // Let user know that it's done
-                form.find('input.btn').addClass('btn-success').val('Successfully Submitted!');
-                setTimeout(function() {
-                    form.find('input.btn').removeClass('btn-success').addClass('btn-primary').val('SIGN UP');
-                }, 5000);
-                // Reset the form for next submission
-                form.trigger('reset');
-            } else {
-                // Let user know that it failed
-                // Keep the form as is for resubmission
-                form.find('input.btn').addClass('btn-warning').val('Error. Try again?');
-                setTimeout(function() {
-                    form.find('input.btn').removeClass('btn-warning').addClass('btn-primary').val('SIGN UP');
-                }, 5000);
-            }
+        var handle_success = function(form) {
+            // Let user know that it's done
+            var button = form.find('input.btn')
+            var sign_up_text = button.val()
```

Comment:
Believe you can move the button and sign_up_text variable declarations up to where you define the form var (would reduce code duplication and protect against possible race-condition when reading/setting button text).